testsuite/test_ta_workflow.py

```diff
@@ -768,3 +768,30 @@ def test_ta_logout(client, STC, base_db_setup):
 
     response = client.post('/logout/')
     STC.assertRedirects(response, '/')
+
+
+@pytest.mark.django_db
+def test_ta_reveal(client, STC, base_db_setup):
+    (_, ta, user, course, assignment) = base_db_setup
+
+    client.login(username=ta.username, password='pw')
+
+    response = client.get(f'/assignment/{course.pk}/{assignment.pk}/reveal')
+
+    STC.assertRedirects(response, f'/assignment/{course.pk}/{assignment.pk}')
+
+    if not assignment.hidden_score:
+        assert False
```

Comment:
`assert assignment.hidden_score == False` is cleaner

Reply:
This one generates a warning on flake8